internal/isoeditor/rhcos.go

```diff
@@ -96,6 +104,33 @@ func (e *rhcosEditor) CreateClusterMinimalISO(ignition string, staticIPConfig st
 	return e.create()
 }
 
+func (e *rhcosEditor) embedInitrdPlaceholders() error {
+	// Create ramdisk image placeholder
+	if err := e.createImagePlaceholder(ramDiskImagePath); err != nil {
+		return errors.Wrap(err, "Failed to create placeholder for custom ramdisk image")
+	}
+
+	// ToDo: store offsets in system area
+
+	return nil
+}
+
+func (e *rhcosEditor) createImagePlaceholder(imagePath string) error {
+	f, err := os.Create(e.isoHandler.ExtractedPath(imagePath))
+	if err != nil {
+		return err
+	}
+	defer f.Close()
+
+	data := make([]byte, initrdPaddingLength)
+	_, err = f.Write(data)
```

Comment:
I think `f.Truncate(initrdPaddingLength)` would be better than maintaining a 256k array in memory.

Reply:
right, should be good enough.